src/components/componentDiff/ComponentDiffTab.js

```diff
@@ -219,6 +221,65 @@ const ComponentVersionDiff = ({
     rightVersion: rightVersion,
   })
 
+  const HandleExportDiff = () => {
+    const addedResources = []
+    const removedResources = []
+
+    diff.components_changed?.forEach(component => {
+
+      const formatResourceName = (resource, component) => {
+        if (resource.type === 'ociImage') {
+          return `[Docker] ${component.name.split('/', 1)[0]}/${resource.name}`
+        }
+        return `${component.name.split('/', 1)[0]}/${resource.name}`
+      }
+
+      component.right.resources?.added?.forEach(resource => {
+        addedResources.push(formatResourceForYaml(component.right, resource, formatResourceName))
+      })
+
+      component.right.resources?.removed?.forEach(resource => {
+        removedResources.push(formatResourceForYaml(component.right, resource, formatResourceName))
+      })
+
+
+      component.right.resources?.changed?.forEach(change => {
+        addedResources.push(formatResourceForYaml(component.right, change.to, formatResourceName))
+        removedResources.push(formatResourceForYaml(component.right, change.from, formatResourceName))
+      })
+
+    })
+
+    downloadYaml(Yaml.dump(addedResources, { lineWidth: 1000 }), 'resource-diff-added.yaml')
+    downloadYaml(Yaml.dump(removedResources, { lineWidth: 1000 }), 'resource-diff-removed.yaml')
+  }
+
+  const formatResourceForYaml = (component, resource, formatResourceName) => {
+    const src_url = component.sources?.[0]?.access?.repoUrl || '<unknown>'
```

Comment:
Please use the main source here (if there is any), and the first source only if there is no main source (or `<unknown>` if there is no source at all).

Reference-coding to determine the main source: https://github.com/gardener/cc-utils/blob/e10378c880c526aa3063473ffa26d6a812497ba9/cnudie/util.py#L260

You can use the label from here: https://github.com/open-component-model/delivery-dashboard/blob/228867b1c2abafaae660a361e806798918d8268e/src/ocm/model.js#L62